components/animationcontainer/overview.md

```diff
@@ -12,7 +12,7 @@ position: 0
 
 The <a href="https://www.telerik.com/blazor-ui/animation-container" target="_blank">Blazor Animation Container component</a> enables you to create messages and popups or expandable containers. It lets you define its animation, size and position, and arbitrary content.
 
->tip If you are looking for an option to create [Notification](slug:notification-overview), [Tooltip](slug:tooltip-overview) or expandable container such as [Drawer](slug:drawer-overview), you may use the dedicated components.
+>tip The `AnimationContainer` animates and renders content in-place and does not have all the features of a true popup. [Compare the abilities of all Telerik Blazor popup components](slug:common-kb-popup-component-comparison) to verify if the [Popup](slug:popup-overview) or [Popover](slug:popover-overview) components are more suitable for your needs. If you are looking for an option to create [Notification](slug:notification-overview), [Tooltip](slug:tooltip-overview) or expandable container such as [Drawer](slug:drawer-overview), you may use the dedicated components.
```

Comment:
Note that 

> "Can" is used to express ability or capability, while "may" is used to express permission or possibility

(Source)

In this case, I would recommend going with `you can use the respective components`.